src/alfred3/element/input.py

```diff
@@ -2051,3 +2051,30 @@ def prepare_web_widget(self):
 
         self.choice_labels = self._determine_scope()
         self.choices = self.define_choices()
+
+
+@inherit_kwargs
+class HiddenInput(InputElement):
+    """
+    Provides a hidden entry field.
+
+    Args:
+        {kwargs}
+
+    Examples:
+        ::
+
+            import alfred3 as al
+            exp = al.Experiment()
+
+            @exp.member
+            class Demo(al.Page):
+                name = "demo"
+
+                def on_exp_access(self):
+                    self += al.HiddenInput(name="hi1", default="fixed")
+
+    """
+
+    base_template = jinja_env.get_template("EmptyElement.html.j2")
+    element_template = jinja_env.get_template("HiddenInputElement.html.j2")
```

Comment:
For the filepaths here, you need to include the full relative path. The `jinja_env` points to the directory `templates`, so you need to use, for example, `html/EmptyElement.html.j2`.

Reply:
Done in commit c50a586eda2b42666af1e15ea10350b87ff18989

Reply:
Nice! :)